tests/integration/utils.go

```diff
@@ -52,7 +52,11 @@ const (
 )
 
 var (
-	defaultPGParameters = cluster.PGParameters{"log_destination": "stderr", "logging_collector": "false"}
+	defaultPGParameters = cluster.PGParameters{
+		"log_destination":   "stderr",
+		"logging_collector": "false",
+		"log_checkpoints":   "on",
```

Comment:
Can't see why we wouldn't have this on. It doesn't hurt us and can be extremely useful.